### PR TITLE
Common changes

### PR DESCRIPTION
**Pull request by shanmydell**
Updated release notes for 

- PowerMax
- PowerStore
- PowerFlex
- PowerScale
- Unity
Updated Installation document for 
- PowerMax
- PowerStore
- PowerFlex
- PowerScale
- Unity
Updated Upgradation document for 
- PowerMax
- PowerStore
- PowerFlex
- PowerScale
- Unity

Updated Index page of 
- CSM
- CSI Driver

Updated Tanzu page to show the support.

### CODE REVIEW

File: content/docs/csidriver/installation/helm/unity.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Install CSI Driver for Unity using this procedure.
 
 *Before you begin*
 
- * You must have the downloaded files, including the Helm chart from the source [git repository](https://github.com/dell/csi-unity) with the command ```git clone https://github.com/dell/csi-unity.git```, ready for this procedure.
+ * You must have the downloaded files, including the Helm chart from the source [git repository](https://github.com/dell/csi-unity) with the command ```git clone -b v2.1.0 https://github.com/dell/csi-unity.git```, ready for this procedure.
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/helm/unity.md:41]: You must have the downloaded files, including the Helm chart from the source [git repository](https://github.com/dell/csi-unity) with the command ```git clone -b v2.1.0 https://github.com/dell/csi-unity.git```, as a prerequisite for running this procedure.

File: content/docs/csidriver/upgradation/drivers/isilon.md [hunk 2]
```diff
@@ -8,18 +8,18 @@ Description: Upgrade PowerScale CSI driver
...
 You can upgrade the CSI Driver for Dell EMC PowerScale using Helm or Dell CSI Operator.
 
-## Upgrade Driver from version 1.6.0 to 2.0.0
+## Upgrade Driver from version 2.0.0 to 2.1.0
 
 **Note:** While upgrading the driver via helm, controllerCount variable in myvalues.yaml can be at most one less than the number of worker nodes.
 
 **Steps**
-1. Verify that all pre-requisites to install CSI Driver for Dell EMC PowerScale version 2.0.0 are fulfilled. Note that change in secret format should be taken care.
+1. Verify that all pre-requisites to install CSI Driver for Dell EMC PowerScale version 2.1.0 are fulfilled. Note that change in secret format should be taken care.
```
Comment (rsedlock1958) [inline content/docs/csidriver/upgradation/drivers/isilon.md:16]: Verify that all pre-requisites to install CSI Driver for Dell EMC PowerScale version 2.1.0 are fulfilled. Note that change in secret format should be implemented.

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. There were a couple of minor edits.

File: content/docs/csidriver/upgradation/drivers/isilon.md (hunk 2 above)
Comment (shanmydell) [reply content/docs/csidriver/upgradation/drivers/isilon.md]: @rsedlock1958 : Addressed

File: content/docs/csidriver/installation/helm/unity.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/helm/unity.md]: @rsedlock1958 : Addressed

File: content/docs/_index.md [hunk 3]
```diff
@@ -24,8 +24,8 @@ CSM is made up of multiple components including modules (enterprise capabilities
 | Observability | 1.0 | - | - | - |
 | Replication | 1.0 | - | - | - |
```
Comment (hoppea2) [inline content/docs/_index.md:25]: Authorization will be revved up to 1.1

File: content/docs/FAQ/_index.md [hunk 4]
```diff
@@ -33,13 +33,13 @@ The umbrella repository for every Dell Container Storage Module is: [https://git
...
+| CSM Module        | CSI PowerFlex v2.1 | CSI PowerScale v2.1 | CSI PowerStore v2.1 | CSI PowerMax v2.1 | CSI Unity XT v2.1    |
 | ----------------- | -------------- | --------------- | --------------- | ------------- | --------------- |
-| Authorization v1.0| ✔️              | ✔️               | ❌              | ✔️             | ❌            |
+| Authorization v1.1| ✔️              | ✔️               | ❌              | ✔️             | ❌            |
 | Observability v1.0| ✔️              | ❌              | ✔️               | ❌            | ❌            |
-| Replication   v1.0| ❌             | ❌              | ✔️               | ✔️             | ❌            |
+| Replication   v1.1| ❌             | ❌              | ✔️               | ✔️             | ❌            |
 | Resilency     v1.0| ✔️              | ❌              | ❌              | ❌            | ✔️             |
-| CSM Installer v1.0| ✔️              | ✔️               | ✔️               | ✔️             | ✔️             |
+| CSM Installer v1.1| ✔️              | ✔️               | ✔️               | ✔️             | ✔️             |
```
Comment (hoppea2) [inline content/docs/FAQ/_index.md:43]: CSM Installer will not be revved. We should probably drop it from this table

File: content/docs/csidriver/_index.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ The CSI Drivers by Dell EMC implement an interface between [CSI](https://kuberne
 | CentOS        |     7.8, 7.9     |      7.8, 7.9       |     7.8, 7.9     |      7.8, 7.9     |     7.8, 7.9     |
 | SLES          |        15SP2        |        15SP2        |       15SP2      |         15SP2     |       15SP2      |
```
Comment (mjsdell) [inline content/docs/csidriver/_index.md:23]: Need to change to SP3

Comment (mjsdell) [reply content/docs/csidriver/_index.md:23]: In this table we should also update the access modes to add RWOP for the drivers that implemented it.
@shanmydell please double check the pmax microcode/unisphere versions specified in the Support storage platforms section.

File: content/docs/csidriver/release/powerscale.md [hunk 6]
```diff
@@ -3,20 +3,15 @@ title: PowerScale
...
-- Added the ability to enable/disable installation of snapshotter sidecar with driver installation.
-- Added support to make dnsPolicy of node component configurable via Operator as well.
-- Added the ability to configure kubelet directory path.
-- Updated support for dynamic logging configuration.
+
+- Added support for OpenShift v4.9.
+- Added support for CSI spec 1.5.
+- Added support for new access modes in CSI Spec 1.5.
+- Added support for PV/PVC metrics.
+- Added ability to accept leader election timeout flags.
```
Comment (randeepsharma) [inline content/docs/csidriver/release/powerscale.md:14]: Do we need to add the health monitor feature as part of release for all drivers that implemented it?

[the remark this answers is not in the document]
Comment (randeepsharma) [reply content/docs/csidriver/release/powerscale.md:14]: Dell EMC PowerScale 9.3

File: content/docs/FAQ/_index.md (hunk 4 above)
Comment (shanmydell) [reply content/docs/FAQ/_index.md:43]: addressed

File: content/docs/csidriver/_index.md (hunk 5 above)
Comment (shanmydell) [reply content/docs/csidriver/_index.md]: addressed

File: content/docs/csidriver/release/powerscale.md (hunk 6 above)
Comment (shanmydell) [reply content/docs/csidriver/release/powerscale.md:14]: addressed

[the remark this answers is not in the document]
Comment (shanmydell) [reply content/docs/csidriver/release/powerscale.md:14]: Addressed

**Review by randeepsharma — APPROVED** [review]
LGTM. Good work @shanmydell

File: content/docs/csidriver/release/unity.md [hunk 7]
```diff
@@ -3,17 +3,17 @@ title: Unity
...
-- Added support for Openshift 4.8.
-- Added the ability to change log level and log format of CSI driver and change them dynamically.
-- Added the ability to configure kubelet directory path.
-- Added the ability to enable/disable installation of resizer sidecar with driver installation.
-- Added the ability to enable/disable installation of snapshotter sidecar with driver installation.
-- Added support for consistent config parameters across CSI drivers.
+- Added support for OpenShift v4.9.
+- Added support for CSI spec 1.5.
+- Added support for new access modes in CSI Spec 1.5.
+- Added support for PV/PVC metrics.
```
Comment (rajendraindukuri) [inline content/docs/csidriver/release/unity.md:13]: @shanmydell "volume health monitoring" covers pvc metrics part. So we can remove the following statements here
- Added support for PV/PVC metrics.
- Added NFS support for PVC metrics in controller.

Comment (mjsdell) [reply content/docs/csidriver/release/unity.md:13]: Well perhaps we just simplify to 'Added support for PV/PVC metrics' as even without monitor enabled there are still metrics gathering, the monitor just adds to that.

**Review by francis-nijay — APPROVED** [review]

**Review by rajendraindukuri — APPROVED** [review]

**Review by karthikk92 — APPROVED** [review]

**Pull request merged**